amadeusgpt/main.py

```diff
@@ -32,8 +32,13 @@
     Object,
     Orientation,
     Database,
+    AnimalSeq,
+    MABE,
+    Segmentation,
+    MausHausSeg,
+    SAM,
 )
-from amadeusgpt.modules.implementation import *
+#from amadeusgpt.modules.implementation import *
```

Comment:
The reason I do import * here is to support dynamic loading. So your IDE won't detect any usage of those python files but they might be used in the runtime. Is there any reason we want to remove this?

Reply:
I opted to import directly all the classes vs import * which is typically a bad idea

https://google.github.io/styleguide/pyguide.html

but if this does not work for amadeus for reasons I don't fully know, we can revert. I actually didn't mean to push this change on this branch (also bad form from me!)